src/snapred/backend/dao/state/GroupingMap.py

```diff
@@ -0,0 +1,90 @@
+import os
+from pathlib import Path
+from typing import Dict, List
+
+from pydantic import BaseModel
+
+# import json
+from snapred.backend.dao.state.FocusGroup import FocusGroup
+from snapred.backend.log.logger import snapredLogger
+from snapred.meta.Config import Config, Resource
+
+logger = snapredLogger.getLogger(__name__)
+
+
+class GroupingMap(BaseModel):
+    availableGroups: Dict[str, Dict[str, Path]] = {}
+    stateID: str = None
+
+    @property
+    def lite(self) -> Dict[str, Path]:
+        return self.availableGroups["lite"]
+
+    @property
+    def native(self) -> Dict[str, Path]:
+        return self.availableGroups["native"]
```

Comment:
The intention here is to be able to access the group path as a dictionary value so that we can just call `lite["name"]` to get the path. The dictionary is formatted as `FocusGroup->Name : FocusGroup->Definition`